cli/ops/jupyter.rs

```diff
@@ -50,17 +52,28 @@ pub async fn op_jupyter_broadcast(
 
   let maybe_last_request = last_execution_request.borrow().clone();
   if let Some(last_request) = maybe_last_request {
-    (*iopub_socket.lock().await)
-      .send(
-        &last_request
-          .new_message(&message_type)
-          .with_content(content)
-          .with_metadata(metadata)
-          .with_buffers(
-            buffers.into_iter().map(|b| b.to_vec().into()).collect(),
-          ),
-      )
-      .await?;
+    let content = JupyterMessageContent::from_type_and_content(
+      &message_type,
+      content.clone(),
+    )
```

Comment:
If we want to, we could send the `content` as a raw value. This does enforce that a message is valid before sending it over the wire if the message type is known though.

Reply:
I think this approach is fine. Should we surface error to the frontend of the notebook though?